pkg/server/admin_test.go

```diff
@@ -400,6 +400,12 @@ func TestAdminAPITableDetails(t *testing.T) {
 	testAdminAPITableDetailsInner(t, "test", "tbl")
 }
 
+// #14056
```

Comment:
I don't think this is going to pass our linter - function comments in go need to be of the form `// [ObjectBeingCommented] [description]`, or must not be present at all.  In this case, that would mean the comment has to be of a form similar to `// TestAdminAPITableDetailsUppercase ensures the Admin API works correctly with uppercase table names. This is a regression of #14056`.

If you don't want to be quite so verbose, you could also just put the comment inside the body of the function.  However, even then the comment should be slightly more descriptive like "// Regression test for issue #14056", the issue number without context is not that helpful - it needs to be distinguished from an active issue or pull request.